src/test/java/uk/gov/companieshouse/web/lfp/controller/lfp/EnterLFPDetailsControllerTest.java

```diff
@@ -128,8 +128,6 @@ void postRequestCompanyNumberLowerCase() throws Exception {
                 .param(PENALTY_NUMBER_ATTRIBUTE, VALID_PENALTY_NUMBER)
                 .param(COMPANY_NUMBER_ATTRIBUTE, LOWER_CASE_LLP))
                 .andExpect(status().is3xxRedirection())
-                .andExpect(model().attributeExists(TEMPLATE_NAME_MODEL_ATTR))
```

Comment:
These attribute checks have been removed and I notice it's after a redirect. Are you sure we can remove these, and that we don't instead need to update the controllers to use [flash attributes](https://docs.spring.io/spring-framework/reference/web/webmvc/mvc-controller/ann-methods/flash-attributes.html) ?

Reply:
Yes , we need these attribute and I have added again and with change in controller to add flush attributes. Thank you